src/app/api/lookup/bulk/route.ts

```
@@ -52,13 +56,25 @@ export async function POST(request: NextRequest) {
           };
         }
 
-        // Make the API call
-        const response = await fetch(`${baseUrl}${apiEndpoint}`, {
+        // Call the route handler directly instead of performing a loopback HTTP request,
+        // which prevents Host-header Server-Side Request Forgery (SSRF) vulnerabilities.
+        const mockRequest = new NextRequest(new URL("http://localhost" + apiEndpoint), {
```

ESLint

[error] Unexpected string concatenation.

(prefer-template)